packages/backend/src/handlers/actions/quests/createCompletion/createCompletion.ts

```diff
@@ -0,0 +1,67 @@
+import { CreateQuestCompletionInput, CreateQuestCompletionOutput,
+  Quest_Completion_Insert_Input,
+  QuestRepetition_Enum,
+  QuestStatus_Enum, } from '../../../../lib/autogen/hasura-sdk';
+import { client } from '../../../../lib/hasuraClient';
+
+export async function createCompletion(
+  playerId: string,
+  questCompletion: CreateQuestCompletionInput,
+): Promise<CreateQuestCompletionOutput> {
+  if (!questCompletion.submission_link && !questCompletion.submission_text) {
+    throw new Error('Must provide at least a submission link or text');
+  }
```

Comment:
Since the submission_link and submission_text fields are required in the mutation, do we even need to check for this?

Reply:
Mistake, they should be both mandatory in the mutation